lib/update/cluster/phases/kubernetes.go

```diff
@@ -380,96 +378,4 @@ func removeKubeletPermissions(client *kubeapi.Clientset) error {
 	return nil
 }
 
-// supportsTaints determines if the specified gravity package
-// supports node taints.
-func supportsTaints(gravityPackage loc.Locator) (supports bool, err error) {
-	ver, err := gravityPackage.SemVer()
-	if err != nil {
-		return false, trace.Wrap(err)
-	}
-	return defaults.BaseTaintsVersion.Compare(*ver) <= 0, nil
-}
-
-func waitForEndpoints(ctx context.Context, client corev1.CoreV1Interface, server storage.Server) error {
```

Comment:
This has moved to lib/update/utils.go as this is shared between multiple packages that also do rolling update.